proto/aep-api/aep/api/operation.proto

```diff
@@ -34,7 +37,80 @@ extend google.protobuf.MethodOptions {
   // long-running operations.
   //
   // Required for methods that return `aep.api.Operation`; invalid otherwise.
-  aep.api.OperationInfo operation_info = 1049;
+  // see https://github.com/protocolbuffers/protobuf/pull/20839
+  aep.api.OperationInfo operation_info = 1264;
+}
+
+// Manages long-running operations with an API service.
+//
+// When an API method normally takes long time to complete, it can be designed
+// to return [Operation][aep.api.Operation] to the client, and the
+// client can use this interface to receive the real response asynchronously by
+// polling the operation resource, or pass the operation resource to another API
+// (such as Pub/Sub API) to receive the response.  Any API service that returns
+// long-running operations should implement the `Operations` interface so
+// developers can have a consistent client experience.
+service OperationsService {
+
+  // Lists operations that match the specified filter in the request. If the
+  // server doesn't support this method, it returns `UNIMPLEMENTED`.
+  rpc ListOperations(ListOperationsRequest) returns (ListOperationsResponse) {
+    option (google.api.http) = {
+      get: "/v1/{name=operations}"
+    };
+    option (google.api.method_signature) = "name,filter";
+  }
+
+  // Gets the latest state of a long-running operation.  Clients can use this
+  // method to poll the operation result at intervals as recommended by the API
+  // service.
+  rpc GetOperation(GetOperationRequest) returns (Operation) {
+    option (google.api.http) = {
+      get: "/v1/{name=operations/**}"
+    };
+    option (google.api.method_signature) = "name";
+  }
+
+  // Deletes a long-running operation. This method indicates that the client is
```

Comment:
It seems to me that deleting an operation should be the same as canceling it. 

Otherwise, deleting the operation seems like a strange action for a user to take.

Reply:
agreed it's a weird one - maybe @rofrankel has an opinion?

I'm leaning toward keeping the RPC, but leaving the guidance that it can be unsupported and just return "UNIMPLEMENTED". In our case, as you mentioned, I think we'd want to use `Cancel`